x-pack/plugins/security_solution/public/actions/filter/cell_action/filter_in.test.ts

```diff
@@ -5,18 +5,16 @@
  * 2.0.
  */
 
-import { createFilterManagerMock } from '@kbn/data-plugin/public/query/filter_manager/filter_manager.mock';
 import { createMockStore, mockGlobalState } from '../../../common/mock';
 import { createFilterInCellActionFactory } from './filter_in';
 import type { SecurityCellActionExecutionContext } from '../../types';
-import { createStartServicesMock } from '../../../common/lib/kibana/kibana_react.mock';
 import { TableId } from '@kbn/securitysolution-data-table';
 import { TimelineId } from '../../../../common/types';
 import { KBN_FIELD_TYPES } from '@kbn/field-types';
+import { createServicesMock } from './test/utils';
 
-const services = createStartServicesMock();
+const services = createServicesMock();
```

Comment:
Why not add the `timelineFilterManager` service mock in the regular `createStartServicesMock`? We don't usually create a new mocking function every time we add a new service.

Reply:
sure!